src/aat/resources/features/F-044/S-579_Later_Case_Update_By_Solicitor_3.td.json

```diff
@@ -0,0 +1,175 @@
+{
+  "title": "Update the case by Solicitor",
+  "_extends_": "S-579-Update_Case_For_Solicitor_Data_Base",
+  "_guid_": "S-579_Later_Case_Update_By_Solicitor_3",
+  "specs": [
+    "to update the same case by Solicitor 3"
+  ],
+  "user": {
+    "username": "befta.caseworker.2.solicitor.3@gmail.com",
+    "password": "Pa55word11"
+  },
+  "request": {
+    "pathVariables": {
+      "cid": "${[scenarioContext][parentContext][childContexts][Befta_Jurisdiction2_Default_Full_Case_Creation_Data][testData][actualResponse][body][id]}"
+    },
+    "body": {
+      "data": {
+        "DocumentField2": {
+          "document_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943",
+          "document_binary_url": "http://dm-store-aat.service.core-compute-aat.internal/documents/ae5c9e4b-1385-483e-b1b7-607e75dd3943/binary",
+          "document_filename": "Elastic Search test Case.png --> updated by Solicitor 3"
+        }
+      },
+      "event_token": "${[scenarioContext][parentContext][childContexts][S-579-Prerequisite_Solicitor_3_Token_For_Update_Case][testData][actualResponse][body][token]}"
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 201,
+    "responseMessage": "Created",
+    "headers": {
+      "Content-Encoding": "gzip",
+      "vary": "accept-encoding"
+    },
+    "body": {
+      "id": "[[ANY]]",
+      "jurisdiction": "BEFTA_JURISDICTION_2",
+      "state": "TODO",
+      "version": 2,
+      "case_type_id": "BEFTA_CASETYPE_2_1",
+      "created_date": "[[ANY]]",
+      "last_modified": "[[ANY]]",
+      "security_classification": "PUBLIC",
+      "case_data": {
```

Comment:
Can use inheritance.

Reply:
No, we can't use as the data is different